integrations/docs/Integration-plugin-tasks.md

```diff
@@ -19,7 +19,29 @@ template mapping for each catalog schema entity. This will allow the future Inte
 
 These mapping specify a backed `data-stream` index pattern they conform with [Naming Pattern](observability/Naming-convention.md).
 
-**_In the future this catalog may be exposed using a dedicated REST API_**
+**API**
+The catalog API can be queries according to the next fields:
+ - type
+ - catalog
+ - category
+ - version
+
+`GET _integration/catalog?filter=type:Logs&catalog:Observability&category:web`
+
+ This call will result with the appropriate index_template IDs corresponding to the query:
+
+```json
```

Comment:
It's worth noting that Github supports [json5](https://json5.org/) language formatting, not important to change now but for future reference setting `json5` will avoid the red syntax errors.

Reply:
cool - thanks